controllers/operator.go

```diff
@@ -45,13 +47,20 @@ type operator struct {
 	// XXX: should it be a part of mp?
 	metricsUpdated bool
 
-	missingIds          []int32
-	pausedIds           []int32
-	runningIds          []int32
-	laggingIds          []int32
-	toRemoveInstances   []*appsv1.Deployment
-	toUpdateInstances   []*appsv1.Deployment
-	toEstimateInstances []*appsv1.Deployment
+	// partition assignments, indicates relation of consumerId to partitions
+	// these arrays are operate on deployments, not partitions
+	assignments [][]int32
```

Comment:
Can we have this be always sorted?

Reply:
yes, it should be sorted, thanks!

Reply:
it is always sorted, because its populated from the range generator in https://github.com/lwolf/konsumerator/pull/28/files#diff-6951588dc1b9c9530cc7d601096a2f6cR142